### PR TITLE
Add default CNI configuration

### PR DESCRIPTION
**Pull request by baude**
podman needs a pair of configuration files to set up its default
network configuration: a bridge and loopback file.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
@baude I think you forgot to check in the new CNI configs

File: Dockerfile [hunk 1]
```diff
@@ -85,16 +85,16 @@ RUN set -x \
...
-RUN set -x \
-       && export GOPATH="$(mktemp -d)" \
-       && git clone https://github.com/runcom/plugins.git "$GOPATH/src/github.com/containernetworking/plugins" \
-       && cd "$GOPATH/src/github.com/containernetworking/plugins" \
-       && git checkout -q "$CNI_TEST_BRANCH" \
-       && ./build.sh \
-       && mkdir -p /opt/cni/bin \
-       && cp bin/bridge /opt/cni/bin/bridge-custom \
-       && rm -rf "$GOPATH"
+#ENV CNI_TEST_BRANCH custom-bridge
```
Comment (rhatdan) [inline Dockerfile]: Should we just remove these?  Why comment them out?

File: test/helpers.bash [hunk 2]
```diff
@@ -104,7 +104,7 @@ if [ -e /usr/sbin/selinuxenabled ] && /usr/sbin/selinuxenabled; then
     filelabel=$(awk -F'"' '/^file.*=.*/ {print $2}' /etc/selinux/${SELINUXTYPE}/contexts/lxc_contexts)
     chcon -R ${filelabel} $TESTDIR
 fi
-LIBPOD_CNI_CONFIG="$TESTDIR/cni/net.d/"
+LIBPOD_CNI_CONFIG="/etc/cni/net.d/"
```
Comment (rhatdan) [inline test/helpers.bash]: Why are you installing directly into the system?  Shouldn't this stay in the TESTDIR?

File: Dockerfile (hunk 1 above)
Comment (baude) [reply Dockerfile]: Yup ... I just wanted to make sure I was right before deleting it ... Now deleted.

File: test/helpers.bash (hunk 2 above)
Comment (baude) [reply test/helpers.bash]: With papr and travis being so different, we need to commit to installing stuff to the system so tests can be done similarily or we will be forever chasing our tails -- imho

**Comment by baude** [comment]
@rhatdan should be good to go now

File: contrib/cni/README.md [hunk 3]
```diff
@@ -1,16 +0,0 @@
-## `contrib/cni` ##
```
Comment (mheon) [inline contrib/cni/README.md]: Do we need to remove this? It seems like a helpful pointer to where to get the plugins, if nothing else.

File: test/helpers.bash (hunk 2 above)
Comment (rhatdan) [reply test/helpers.bash]: I don't think we should be doing this, why can't we just install the two files into the $TESTDIR/cni/net.d from the helpers.bash script?  If we use the system path, we should not install to it, and just require that these packages be installed.  

I am fearful that we will override some defaults that a user is actually using.

File: contrib/cni/README.md (hunk 3 above)
Comment (baude) [reply contrib/cni/README.md]: imho, they should be installing a containersnetworking-cni package.  we have to get away from catering to installation from source.

**Comment by baude** [comment]
@rhatdan ok, back to installing into TESTDIR, ready for merge

**Comment by baude** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**